packages/web-components/src/divider/divider.docs.mdx

```diff
@@ -0,0 +1,64 @@
+import { Meta, Story } from '@storybook/addon-docs';
```

Comment:
Why is this file an mdx? Is is supposed to be visible in the storybook?
Or is it just a spec? In FUIR9 we use `docs/spec.md` for component specs.

Reply:
Removing this file as info from this doc is present in the storybook args table under the Description and Default headings.